translations/ja.json

```
@@ -864,55 +864,55 @@
 	"[{{.id}}] {{.msg}} {{.error}}": "[{{.id}}] {{.msg}} {{.error}}",
-	"\\\"minikube cache\\\" will be deprecated in upcoming versions, please switch to \\\"minikube image load\\\"": "",
+	"\\\"minikube cache\\\" will be deprecated in upcoming versions, please switch to \\\"minikube image load\\\"": "「minikube cache」は今後のバージョンで廃止予定になりますので、「minikube image load」に切り替えてください",
 	"adding node": "ノードを追加しています",
-	"addon '{{.name}}' is currently not enabled.\nTo enable this addon run:\nminikube addons enable {{.name}}": "「 {{.name}} 」アドオンは現在無効になっています。\n有効にするためには、以下のコマンドを実行してください。 \nminikube addons enable {{.name}}",
-	"addon '{{.name}}' is not a valid addon packaged with minikube.\nTo see the list of available addons run:\nminikube addons list": "「 {{.name}} 」アドオンは minikube では有効なアドオンではありません。\n利用可能なアドオンの一覧を表示するためには、以下のコマンドを実行してください。 \nminikube addons list",
-	"addons modifies minikube addons files using subcommands like \"minikube addons enable dashboard\"": "addons では以下のようにサブコマンドを使用することで、 minikube のアドオンのファイルを編集することができます。 \"minikube addons enable dashboard\"",
-	"auto-pause addon is an alpha feature and still in early development. Please file issues to help us make it better.": "",
-	"bash completion failed": "bash の補完が失敗しました",
-	"bash completion.": "",
-	"call with cleanup=true to remove old tunnels": "cleanup=true で呼び出すことで、古い tunnel を削除することができます",
-	"cancel any existing scheduled stop requests": "",
-	"cannot specify --kubernetes-version with --no-kubernetes,\nto unset a global config run:\n\n$ minikube config unset kubernetes-version": "",
-	"config modifies minikube config files using subcommands like \"minikube config set driver kvm\"\nConfigurable fields:\\n\\n": "config では以下のようにサブコマンドを使用して、minikube の設定ファイルを編集することができます。 \"minikube config set driver kvm\"\n設定可能なフィールドは以下です。\\n\\n",
-	"config modifies minikube config files using subcommands like \"minikube config set driver kvm2\"\nConfigurable fields: \\n\\n": "",
-	"config view failed": "設定を表示するのに失敗しました",
-	"containers paused status: {{.paused}}": "",
-	"dashboard service is not running: {{.error}}": "ダッシュボードのサービスが動いていません。 {{.error}}",
+	"addon '{{.name}}' is currently not enabled.\nTo enable this addon run:\nminikube addons enable {{.name}}": "'{{.name}}' アドオンは現在無効になっています。\n有効にするためには、以下のコマンドを実行してください。 \nminikube addons enable {{.name}}",
+	"addon '{{.name}}' is not a valid addon packaged with minikube.\nTo see the list of available addons run:\nminikube addons list": "'{{.name}}' は minikube にパッケージングされた有効なアドオンではありません。\n利用可能なアドオンの一覧を表示するためには、以下のコマンドを実行してください。 \nminikube addons list",
+	"addons modifies minikube addons files using subcommands like \"minikube addons enable dashboard\"": "addons コマンドは「minikube addons enable dashboard」のようなサブコマンドを使用することで、 minikube アドオンファイルを修正します",
+	"auto-pause addon is an alpha feature and still in early development. Please file issues to help us make it better.": "auto-pause アドオンはアルファ機能で、未だに開発の初期段階です。auto-pause アドオン改善の手助けのために、問題は報告してください。",
```

```suggestion
	"auto-pause addon is an alpha feature and still in early development. Please file issues to help us make it better.": "auto-pause アドオンはアルファ機能で、まだ開発の初期段階です。auto-pause アドオン改善の手助けのために、問題は報告してください。",
```
「未だに」は意味が少し強く感じたため、「まだ」にしたらどうでしょうか。